docs/source/user-guide/vision/quickstarts/plot_segmentation_tutorial.py

```diff
@@ -27,151 +27,108 @@
 # %%
 # Defining the data and model
 # ===========================
+# .. note::
+#   In this tutorial, we use the pytorch to create the dataset and model. To see how this can be done using tensorflow
+#   or other frameworks, please visit the :ref:`creating VisionData guide <vision_data__creating_vision_data>`.
 
-# Importing the required packages
-import matplotlib.pyplot as plt
-import numpy as np
-import torch
-import torchvision.transforms.functional as F
-
-
-# %%
 # Load Data
 # ~~~~~~~~~
-# The model in this tutorial is used to detect different objects in images (labels based on the Pascal VOC dataset).
+# The model in this tutorial is used to detect different object segments in images (labels based on the Pascal VOC dataset).
 # The model is trained to identify 20 different objects (person, bicycle etc.) and background.
 # The dataset itself is the COCO128 dataset with semantic segmentation labels, mapped to the Pascal VOC labels
-# (Originally, the COCO dataset includes more labels, but those has been filtered out)
-# The dataset can be loaded as a pytorch DataLoader object from deepchecks.vision.datasets.segmentation, as is done in
-# this tutorial, but can also be loaded as a SegmentationData object
+# (Originally, the COCO dataset includes more labels, but those have been filtered out).
+# The dataset can be loaded as a pytorch Dataset object from deepchecks.vision.datasets.segmentation, as is done in
+# this tutorial, but can also be loaded as a Vision object using the "load_dataset" function from that directory,
 
 # The full pascal VOC data and information can be found here: http://host.robots.ox.ac.uk/pascal/VOC/voc2012/
 # And the COCO128 dataset can be found here: https://www.kaggle.com/datasets/ultralytics/coco128
 
-from torchvision.utils import draw_segmentation_masks
+from deepchecks.vision.datasets.segmentation.segmentation_coco import CocoSegmentationDataset, load_model
 
-from deepchecks.vision.datasets.segmentation.segmentation_coco import load_dataset, load_model
-from deepchecks.vision.segmentation_data import SegmentationData
-
-train_loader = load_dataset(object_type='DataLoader', train=True)
-test_loader = load_dataset(object_type='DataLoader', train=False)
+train_dataset = CocoSegmentationDataset.load_or_download(train=True)
+test_dataset = CocoSegmentationDataset.load_or_download(train=False)
```

Comment:
can add some prints (similar to other tutorials)

Reply:
Done